e2e/terraform/network.tf

```diff
@@ -0,0 +1,78 @@
+data "aws_vpc" "default" {
+  default = true
+}
+
+resource "aws_security_group" "primary" {
+  name   = "${local.random_name}"
+  vpc_id = "${data.aws_vpc.default.id}"
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  # Nomad
+  ingress {
+    from_port   = 4646
+    to_port     = 4646
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
I feel like this potentially opens us up to a security issue, especially if resources don't get de-provisioned. I'm not sure if we should take action today, but we may want to have a ticket/todo to use mutual tls?